apps/web/lang/de.json

```
@@ -0,0 +1,359 @@
+      "personalData": {
+        "yourName": "Ihr Name",
+        "firstName": "Vorname",
+        "lastName": "Nachname",
+        "edit": "Bearbeiten",
+        "change": "Ändern",
+        "contactInformation": "Kontaktinformationen",
+        "yourPassword": "Ihr Passwort",
+        "passwordChange": "Passwort ändern",
+        "newPassword": "Neues Passwort",
+        "newPasswordAgain": "Neues Passwort wiederholen",
+        "changePassword": "Passwort ändern",
+        "currentPassword": "Aktuelles Passwort",
+        "passwordHelp": "Verwenden Sie 8 oder mehr Zeichen mit einer Mischung aus Buchstaben, Zahlen und Symbolen."
+      },
+      "billingDetails": {
+        "billingAddress": "Rechnungsadresse",
+        "edit": "Bearbeiten"
+      },
+      "shippingDetails": {
+        "edit": "Bearbeiten",
+        "shippingAddress": "Lieferadresse"
+      }
+    },
+    "ordersAndReturns": {
+      "heading": "Bestellungen & Retouren",
+      "orderId":"Order-ID",
+      "orderDate":"Auftragsdatum",
+      "amount":"Anzahl",
+      "status":"Status",
+      "details":"Details",
+      "myOrders":"Meine Bestellungen",
+      "noOrders": "Sie haben noch nicht bei uns eingekauft",
+      "continue": "Einkauf fortsetzen",
+      "noOrdersAltText": "Keine Bestellungen",
+      "returnsHeading": "Retouren",
+      "returnsAltText": "Keine Retouren",
+      "returnsDescription": "Sie haben noch keine Produkte zurückgeschickt",
+      "section": {
+        "myOrders": "Meine Bestellungen",
+        "returns": "Retouren"
+      },
+      "orderDetails": {
+        "heading": "Bestellübersicht",
+        "orderId": "Order-ID",
+        "orderDate": "Auftragsdatum",
+        "paymentAmount": "Gesamtsumme",
+        "status": "Status",
+        "price": "Preis",
+        "quantity": "Anzahl",
+        "subtotal": "Zwischensumme",
+        "tableCaption": "Bestellübersicht",
+        "product": "Produkt",
+        "itemsSubtotal": "Produkte Zwischensumme",
+        "delivery": "Versand",
+        "estimatedTax": "MwSt.",
+        "total": "Gesamtsumme", 
+        "billingAddress": "Rechnungsadresse",
+        "paymentMethod": "Zahlungsart",
+        "shippingAddress": "Lieferadresse"
+      }
+    }
+  },
+  "auth": {
+    "login": {
+      "heading": "Login",
+      "submitLabel": "Login",
+      "forgotPasswordLabel": "Passwort vergessen?",
+      "rememberMeLabel": "An mich erinnern",
+      "createAccountBanner": "Zum ersten mal hier? {0}.",
+      "createAccountLinkLabel": "Konto erstellen"
+    },
+    "signup": {
+      "heading": "Konto erstellen",
+      "createButton": "Konto erstellen",
+      "bannerText": "Sie haben bereits ein Konto? Wechseln Sie zu  {login}",
+      "modal": {
+        "heading": "Willkommen",
+        "description": "Ihr Konto wurde erstellt. Sie können jetzt {information} zu Ihrem Konto hinzufügen, damit Sie dies nicht später während Ihres Einkaufs tun müssen.",
+        "information": "Informationen hinzufügen",
```

```suggestion
        "information": "Informationen",
```
If this gets inserted into `description`.